charts/karpenter/values.yaml

```diff
@@ -0,0 +1,79 @@
+nameOverride: ""  # Override the default name, empty means no override
+fullnameOverride: ""  # Override the full name, empty means no override
+
+# -- Global log level, defaults to 'info'
+logLevel: info
+# -- Log outputPaths - defaults to stdout only
+logOutputPaths:
+  - stdout
+# -- Log errorOutputPaths - defaults to stderr only
+logErrorOutputPaths:
+  - stderr
+
+# -- Additional annotations to add into metadata.
+additionalAnnotations: {}  # Empty dictionary for extra annotations
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Automatically mount a ServiceAccount's API credentials?
+  automount: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+podLabels: {}
+
+alibabacloud:
+  profiles:
+    - name: "default"
```

Comment:
The name is `AkProfile`?

Reply:
Its the name of each profile, ali sdk reads multi profile, it's a list for each profile